docs/integrations-guides/rollup-guides/integrations-overview.md

```diff
@@ -1,68 +1,78 @@
+different strategies. We outline the different approaches in this document.
 
-![M0 chain finalization](/img/eigenda/optimistic-M0-dispersal.png)
+## Trusted Integration (Dispersal+Retrieval) {#trusted-integration}
 
-The trusted verification strategy does not do certificate or data verification.
-Instead it focuses on dispersal and retrieval for the sake of simplicity, but at
-the cost of security. This is an integration model fit for testnet. Let's walk
-through the lifecycle of an L2 batch:
+![Insecure Dispersal](../../../static/img/integrations/secure/insecure-dispersal.png)
+
+The trusted integration trusts that the sequencer is verifying certs and 
+posting them to the rollup inbox in a timely fashion. 
+This integration focuses on dispersal and retrieval for the sake of simplicity, 
+but at the cost of security. Let's walk through the lifecycle of an L2 batch:
 
 1. The batcher component of the rollup sequencer prepares an L2 batch, and calls
-the **DisperseBlob()** method on the EigenDA disperser, sending the batch data.
-2. The disperser erasure-encodes the blob into chunks, calculates the KZG
+the **DisperseBlob()** rpc on the EigenDA disperser, sending the batch data.
+1. The disperser erasure-encodes the blob into chunks, calculates the KZG
 commitment, and calculates the KZG proof for each chunk. It then distributes the
 chunks to the EigenDA operator set, where each operator receives a subset of
 the chunks in proportion to its stake. Each operator then stores the chunks its
 received, verifying that each chunk matches its KZG proof and KZG commitment.
 If so, it signs a message certifying that the chunk has been stored and returns
 it to the disperser.
-3. The disperser aggregates the signatures from step 3 into a single BLS
+1. The disperser aggregates the signatures from step 3 into a single BLS
 signature and sends it and some blob metadata to to the EigenDA Manager contract on
 Ethereum. The EigenDA Manager contract on Ethereum is responsible for verifying EigenDA
 certificates, and if they verify, recording that verification in storage.
 Verification consists of ensuring the aggregated signature is valid and is
 based on the current EigenDA operator set. This blob verification status is
 not used in this implementation strategy.
-4. If the sequencer is using the EigenDA disperser, then it shouldn't just trust
+1. If the sequencer is using the EigenDA disperser, then it shouldn't just trust
 the disperser when it says that the blob has successfully been dispersed, it
 should verify by checking onchain. This is important in this integration
 strategy because the rollup inbox does not perform this check. Without this
 check the EigenDA disperser is trusted (in addition to the sequencer).
-5. The batcher then sends a transaction to the rollup inbox contract on
+1. The batcher then sends a transaction to the rollup inbox contract on
```

Comment:
Numbering is off here: every entry is `1.`

There are other numbering issues throughout this doc as well

Reply:
oh yeah weird... think my IDE was doing some weird numbering because of the newlines. I (think I) fixed by indenting the non-numbered lines: [bcd8c47](https://github.com/Layr-Labs/eigenda-docs/pull/72/commits/bcd8c475cf49d8b787fed41cce88c687245d151e)

Reply:
Realized I used some markdown extension to fix numbering in that commit and it numbered the title sections. Removed in [690251e](https://github.com/Layr-Labs/eigenda-docs/pull/72/commits/690251e5231309b130a60a5d318ad6d18358ef1a) and [b490566](https://github.com/Layr-Labs/eigenda-docs/pull/72/commits/b4905660b6341eb8b265b05014e9b8726dcc564b)